.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+Thanks for contributing!
+-->
+
+Some things to make sure you have completed:
+- [ ] Bump Spec version.
+- [ ] Update last modified date.
+- [ ] Update changelog.
+- [ ] Make sure there is generated JSON files from the YAML test files.
```

Comment:
I think this is already covered by the [`json-regenerate-check.yml`](https://github.com/mongodb/specifications/blob/master/.github/workflows/json-regenerate-check.yml) workflow, so perhaps we can just remove this entirely.

```suggestion
- [ ] Make sure there are generated JSON files from the YAML test files.
```